lib/logstash/config/mixin.rb

```diff
@@ -121,13 +123,17 @@ def config_name(name=nil)
       return @config_name
     end
 
+    # Deprecated: Declare the version of the plugin
+    # inside the gemspec.
     def plugin_status(status=nil)
       milestone(status)
     end
 
+    # Deprecated: Declare the version of the plugin
+    # inside the gemspec.
     def milestone(m=nil)
```

Comment:
do we need the parameter here `m=nil`? ... will it be removed after we fix all the calling code?